modules/EnsEMBL/Web/SpeciesDefs.pm

```diff
@@ -656,7 +656,8 @@ sub _expand_database_templates {
         $tree->{'databases'}{'DATABASE_'.$_} = $db_name;
       }
       else {
-        unless ($filename eq 'MULTI' && $SiteDefs::NO_COMPARA) {
+        ## Ignore this step for MULTI, as it may not have a core db
+        unless ($filename eq 'MULTI') {
```

Comment:
We dont use NO_COMPARA anymore?

Reply:
We do, but it's not relevant here and it was causing problems with the covid site which now has compara but no ancestral sequence database.